build.gradle.kts

```diff
@@ -50,12 +63,11 @@ dependencies {
 
     testImplementation("junit:junit:${Versions.junit}")
     testImplementation("org.mockito.kotlin:mockito-kotlin:${Versions.mockitoKotlin}")
-    testImplementation("org.jetbrains:annotations:${Versions.jetbrainsAnnotations}")
 }
 
 java {
     toolchain {
-        languageVersion.set(JavaLanguageVersion.of(8))
+        languageVersion.set(JavaLanguageVersion.of(JavaVersion.VERSION_17.majorVersion))
```

Comment:
I didn't have Java 8  installed, so I thought it makes sense to set recent LTS to be built with, but still target Java 8 (the `options.release` call below)

Reply:
Sounds good